vidarr-pluginapi/src/test/java/ca/on/oicr/gsi/vidarr/InputTypeTest.java

```diff
@@ -0,0 +1,59 @@
+package ca.on.oicr.gsi.vidarr;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public abstract class InputTypeTest {
+  static Map<InputType, String> primitiveTypes =
+      new HashMap<>() {
```

Comment:
There's now the `Map.of` for creating fixed-size maps.

Reply:
Much prettier! Switched.